README.md

```diff
@@ -25,18 +25,28 @@
 
 ## Overview
 
+neo-python is an alternative implementation of the original NEO VM and is easy
+to install/run on *nix environments. For Windows, installing a dependency
+(LevelDB) is more involved, for serious activities, either use a Linux virtual
+machine or run the original NEO VM on Windows.
+
+It comes with batteries included, ie. it runs a full node (syncing and running contracts),
```

Comment:
This part is kind of inaccurate -- you can do a bit more and can also test smart contracts written in any language (but not real debugging). I would just get rid of this full paragraph, and perhaps even the first one too.